### DISCUSSION TITLE
White screen Issue with android emulator

### DISCUSSION THREAD

**Discussion by sajjadalis** [ionic-team/capacitor#5727] [Q&A]
I have a project with Framework7.io and it works fine in the browser but shows a white screen in the Android emulator. Also tested with a freshly created Framework7 project but same issue.

https://i.imgur.com/JVwBv3l.png

These are the errors in debug console in Android studio.
```
E/Capacitor: Unable to read file at path public/plugins
E/Capacitor/BrowserPlugin: Error binding to custom tabs service
```
Complete log: https://sharetext.me/pxobszc79b

Package.json file for the new Framework7 project: https://sharetext.me/cwudmjbbpx

I'm stuck with this for the last few days and can't go forward. I would really appreciate the help. 

I have a project from last year with

- "@capacitor/android”: “^2.4.6”,
- “@capacitor/core”: “^2.4.6”,

And it still works fine in the emulator and the build.

**Comment by jcesarmobile** [comment]
White screen problems are usually caused by javascript errors on the app startup or using an old WebView version
The logs indicate that the emulator is using version 101, so that's very new and it's not the problem then.

If you provide a sample app I can take a look
